### PR TITLE
Add Needs of Morrowind modfiles

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add mods
Add location where possible
Add load order

Comments: list is not exhaustive and there are yet more mods to add (because one found mod means adding it to the list)... anyhow here's this small list for now

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1089,3 +1089,73 @@ plugins:
   - name: 'OAAB_Tel Mora_Female Guards.ESP'
     url: [ 'https://www.nexusmods.com/morrowind/mods/46177/' ]
     after: [ 'OAAB_Tel Mora.esp' ]
+
+  ### Needs of Morrowind ###
+  - name: 'NOM 3.0.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```
Comment (MacSplody) [inline masterlist.yaml]: That link breaks if it has a closing slash, you'll have to make an exception for some links.
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -1089,3 +1089,73 @@ plugins:
...
+    after:
+      - 'NOM3.0.esp'
+      - 'NOM3_PT SO.esp'
+  - name: 'NoM_MC Compatibility Patch.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
+    after:
+      - 'Morrowind Crafting 2-1.esp'
+      - 'NOM3.0.esp'
+      - 'NOM3_PT SO.esp'
+      - 'NoM_Creatures Loot Standard.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Some of these load after rules could be redundant, do the patches not have master files?

Comment (IllusiveMan196) [reply masterlist.yaml]: Not quite, NoM_CoM Compatibility Patch.esp for example only has MW_Children_1_0.esm as master file, except for game+DLCs of course.

File: masterlist.yaml [hunk 3]
```diff
@@ -1089,3 +1089,73 @@ plugins:
...
+      - 'Better Clothes Complete.ESP'
+      - 'Better Clothes_v1.1.esp'
+      - 'Better Clothes_v1.1_nac.esp'
+      - 'Korobal.esp'
+      - 'Morrowind Crafting 2-1.esp'
+      - 'Morrowind Crafting Equipment.esp'
+      - 'Morrowind Crafting Houses.esp'
+      - 'Sentinel+Serendipity.esp'
+  - name: 'NoM_CoM Compatibility Patch.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```

File: masterlist.yaml [hunk 4]
```diff
@@ -1089,3 +1089,73 @@ plugins:
...
+    after:
+      - 'MW_Children_1_0.esm'
+      - 'NOM3.0.esp'
+      - 'NOM3_PT SO.esp'
+      - 'NoM_Creatures Loot Standard.esp'
+      - 'NoM_MC Compatibility Patch.esp'
+  - name: 'NoM_Creatures Loot Standard.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```

File: masterlist.yaml [hunk 5]
```diff
@@ -1089,3 +1089,73 @@ plugins:
...
+    after:
+      - 'NOM3.0.esp'
+      - 'NOM3_PT SO.esp'
+  - name: 'NoM_Creatures Loot PHW.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```

File: masterlist.yaml [hunk 6]
```diff
@@ -1089,3 +1089,73 @@ plugins:
...
+    after:
+      - 'NOM3.0.esp'
+      - 'NOM3_PT SO.esp'
+  - name: 'NoM_MC Compatibility Patch.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```

File: masterlist.yaml (hunk 2 above)
Comment (IllusiveMan196) [reply masterlist.yaml]: \+ quite a lot of patch mods in just my 131 modlist don't have mods they patch as masters, so makes sense to be extra sure here

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks,

**Pull request merged**